### ISSUE TITLE
Type error: step() missing 1 required positional argument: 'epoch'

### ISSUE THREAD

**Issue by itongworld** [bl0/negative-margin.few-shot#8]
Hi, I found a bug here https://github.com/bl0/negative-margin.few-shot/blob/master/lib/utils.py#L77
I found if `epoch` argument does not have a default value like `None`, it would raise a type error in my machine, and models can't be trained, since `_LRScheduler` super class would call `self.step()` in its `__init__` method.

So, have you ever used `GradualWarmupScheduler` scheduler in the pre-training stage? And would this scheduler have a lot effects on the final performance?

Thanks a lot.

**Comment by bl0** [comment]
Hi, thanks for your remind. The bug is introduced after pytorch 1.4, which changes the scheduler API. I have fixed the issue in the [last commit](https://github.com/bl0/negative-margin.few-shot/commit/401ee5c182b0f933b196653c552316a1605868e0).

You can try it again. If there are still problems, please let me know. Thanks.
